setup.py

```diff
@@ -35,9 +35,9 @@
       maintainer_email=MAINTAINER_EMAIL,
       url=URL,
       download_url=DOWNLOAD_URL,
-      install_requires=['numpy>=1.12', 'seaborn>=0.7', 'scipy>=0.18.0',
-                        'matplotlib>=2.0.0', 'Cython>=0.26', 'libsvmdata>=0.3',
```

Comment:
celer.plot_utils.configure_plt() uses them in fact

Reply:
removing the function or the requirements is likely to break existing scripts. So I am not in favor of merging.
We can deprecate the function and remove it and the associated requirements in 0.8 if you want (although I tend to use this function quite a lot in other codes)

Reply:
Alright,

But I'm pretty sure that removing these dependencies won't break the code (otherwise it would have broken down the remote jobs). On the other hand, removing these dependencies would prevent one from downloading and using the examples we exhibit in [the gallery](https://mathurinm.github.io/celer/auto_examples/index.html).

https://github.com/mathurinm/celer/blob/75e3a4c05a067574a7a75925e239310e97110e54/examples/plot_lasso_cv.py#L17

I'm not really with the fact of exposing plotting utils in celer (they don't even show up in the [API](https://mathurinm.github.io/celer/api.html)) as they arbitrary enlarge the scope of the package.